_docs/install-to-azure.md

```diff
@@ -0,0 +1,161 @@
+---
+title: "Install sensenet in Azure"
+source_url: 'https://github.com/SenseNet/sensenet.github.io/blob/master/_docs/install-to-azure.md'
+category: Guides
+version: v7.0
+tags: [install, cloud, azure, app service, sn7]
+description: This article is for developers about installing sensenet in an Azure App Service project.
+---
+
+# Install sensenet in Azure
+
+In this article we provide a step-by-step description of installing sensenet in an [App Service](https://docs.microsoft.com/en-us/azure/app-service/) project in Microsoft Azure.
+
+## Install sensenet locally
+As a prerequisite please install sensenet in Visual Studio using our NuGet packages.
+
+- [Install sensenet](install-sn-from-nuget)
+
+In the following sections we will go through the steps necessary to move this installation to the cloud. In a nutshell we will move the following components to Azure:
+
+- database
+- index
+- messaging
+
+## Deploy the database to the cloud
+We will deploy the database to an **Azure SQL Database**. The easiest way to do that is the following:
+
+1. open SQL Management Studio
+2. right click on the local database
+3. Tasks > **Deploy database to Microsoft Azure SQL Database...**
+
+After connecting to your Azure SQL Server, you'll be able to start the migration process. Depending on the size of your initial database and the network bandwidth to Azure it should finish in a few minutes.
+
+To learn more about the topic, visit [this article](https://docs.microsoft.com/hu-hu/azure/sql-database/sql-database-migrate-your-sql-server-database).
+
+## Install a virtual machine for the Search service
+In a cloud environment sensenet needs a central index (instead of individual local indexes on all app domains). Because the characteristics of the **Lucene search engine** that sensenet uses this requires a **file system storage**, so it needs to be on a virtual machine as of now.
```

Comment:
Not really sure:

```suggestion
In a cloud environment sensenet needs a central index (instead of individual local indexes on all app domains). Because of the characteristics of the **Lucene search engine** that sensenet uses this requires a **file system storage**, so it needs to be on a virtual machine as of now.
```